README.md

```diff
@@ -0,0 +1,222 @@
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- Database Entity Relationship Diagram-->
+
+## 📍 Database Entity Relationship Diagram <a name="database-erd"></a>
+
+- [Link to the Database ERD](https://user-images.githubusercontent.com/20554583/221377937-7bd1e849-ad9b-442d-b7de-50b5cebd52db.png)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- A command line interface
+- Install [Ruby](https://www.ruby-lang.org/)
+- Install [PostgreSQL database](https://www.postgresql.org/)
+- Have Rails set up and running (`gem install rails`)
+
+### Setup
+
+- Clone this repository to your desired folder:
+    ```sh
+    cd my-folder
+    git clone git@github.com:emmiiorji/budget-app.git
+    ```
+- Create a `.env` file in the root directory with your database configuration details following the `.env.example` file as guide
+- Create an account at [cloudinary](https://cloudinary.com/) and include the api keys within the .env file
+
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my_blog
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails db:migrate
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec
+```
+
+<!-- ### Deployment
+
+You can deploy this project using:
+
+```sh
+  
+
+``` -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Emmanuel Orji-Ihuoma**
+
+- GitHub: [@emmiiorji](https://github.com/emmiiorji)
+- Twitter: [@emmiiorji](https://twitter.com/emmiiorji)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/orji-emmanuel)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **Allow transaction to have more than one category**
+- [ ] **Improve user interface**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give a ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [Microverse](https://www.microverse.org/) for the guide
+
```

Comment:
- [ ] `required` 
- [ ]Please make sure that, The  "Acknowledgements" section should give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).